CHANGELOG.md

```diff
@@ -35,6 +35,7 @@
   - Used to filter incoming WebSocket connections: if `Origin` header does not match the list of hosts it will be refused.
   - Value is a comma-separated list of domain names including the port if not `80` nor `443`.
   - Example: `app.some.domain.com,app.other.domain:12345`.
+- [property-view] Added initial version of a selection-based property-view. A default implementation for file selections (via file navigator and default monaco editor).
```

Comment:
minor: the entry will need to be moved under the next release, being `1.10.0`.

Reply:
Oh of course, my bad! I moved this line up  to a new section